attributes.yml

```diff
@@ -217,8 +217,8 @@ exempt_home_users: []
 # main grub boot config file
 grub_main_cfg: '/boot/grub2/grub.cfg'
 
-# superusers for grub boot ( array )
-grub_superusers: ['root']
```

Comment:
I think we should keep this as an array, I have end users that actually have 'superusers' that they don't call root so I want to allow for the tests to work with one or more users defined as 'super'

Reply:
Soooo....it turns out that it's specifically not recommended to use `root`, 'admin`, or `administrator` as your superuser name.

Should that be marked as a warning and/or failure?